CHANGELOG.md

```diff
@@ -5,6 +5,49 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 
+## [1.2.3] - 2021-03-09
+
+### Added
+
+
+### Changed
+
+
+### Fixed
+
+- Fixed `ModelPruning(make_pruning_permanent=True)` pruning buffers getting removed when saved during training ([#6073](https://github.com/PyTorchLightning/pytorch-lightning/pull/6073))
+
```

Comment:
@tchaton Could you remove the extra spaces in Changelog?

Reply:
done